cf_xarray/accessor.py

```diff
@@ -254,9 +254,9 @@ def _get_axis_coord(obj: DataArray | Dataset, key: str) -> list[str]:
         )
 
     search_in = set()
-    if "coordinates" in obj.encoding:
+    if "coordinates" in obj.encoding and obj.encoding["coordinates"]:
```

Comment:
Can we do 
```python
coordinates = obj.encoding.get("coordinates", None)
# Handles case where the coordinates attribute is None
# This is used to tell xarray to not write a coordinates attribute
if coordinates:
    ...
```

and similarly below.

Reply:
Thanks for your feedback. I tried to make the requested adjustments.